Project ToDo list -

expjazz opened the pull request:
### The project has all the features requested:

- You can create projects, and every project contains a list of ToDo items.
- You can edit items, delete, add, create, and delete projects.
- The project is divided into modules, uses ES6 
- You can save your projects in Local Storage

#### Looking forward on how to improve the projects!

Oghenebrume50 commented on src/helpers.js:
```diff
@@ -0,0 +1,161 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable import/no-cycle */
```
This is an indication that there is a circular dependency in your code, this is not good coding practice, please remove this rule and fix the problem, the affected file is the 'Todolistgenerator' 

[see this](https://stackoverflow.com/questions/51094117/dependency-cycle-detected-import-no-cycle)

expjazz commented:
### We made all the requested changes.

- Now we added validations for a valid date. Now you can only create a task or a project in the future.
- Now both Projects and Tasks cannot be valid if they are empty.
- There are no ways to put random text in the dates since we are using the inputs of type DateTime local.

expjazz commented:
## Dear TSE, 

#### We already implemented those validations. What I think happened is that the last TSE checked the wrong branch in his local machine (now we deleted that branch). The branch we wanted to merge and still want is the 'featureV1'. I just tested this feature on my end and it worked. 

![image](https://user-images.githubusercontent.com/61479648/87164769-8d093180-c29f-11ea-99cf-f5eaf5b96c0b.png)
